content/giving/capital.md

```diff
@@ -120,6 +127,13 @@ You can help in a plethora of ways:
 
 **Finally, you can help keep the community active and alive.** Join us at one of the many upcoming [tEply events](../../events/current)!
 
+{{< rawhtml >}}
```

Comment:
Can't you use figure here instead?

https://github.com/alumxi22/website/blob/f36bde7/content/_index.md#L5

Reply:
I couldn't easily find a straightforward hugo-esque way to scale the image so I embedded HTML.

Reply:
The figure and image shortcode provided by the terminal theme allow you to provide arbitrary css (see "style") https://github.com/panr/hugo-theme-terminal#built-in-shortcodes

Reply:
See also: https://laurakalbag.com/processing-responsive-images-with-hugo/

We can create our own shortcodes for handling responsive images: https://gohugo.io/content-management/image-processing/. Good for re-use, and Hugo can handle a lot of things (resizing, cropping, etc.)

Reply:
We could make a shortcode, but this sort of stuff is a lot of overhead to ask people on the board to learn. I'd rather use the less canonical inline HTML because more of the board members are familiar with it. So unless this inline HTML makes the process of helping us run the site a lot harder for you, I'm inclined to stay with it.